src/devices/Mlx90640/Sampling.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Iot.Device.Mlx90640
+{
+    /// <summary>
+    /// MLX90640 sampling
+    /// </summary>
+    public enum Sampling : byte
```

Comment:
see comment about making is ushort and adding directly << 7 in the enum numbers.